docs/sample_config.yaml

```diff
@@ -1973,3 +1973,20 @@ opentracing:
     #
     #  logging:
     #    false
+
+
+## Cache Configuration ##
```

Comment:
I'm surprised the very end of the file is the most appropriate place?

Reply:
I've moved it much further up, near the database config. What do you think?